### PR TITLE
Mumbi/feature/config leaps

### PR DESCRIPTION
**Pull request by Mumbzzz**
config file for leap_left_1 updated to 3Mbps. demo_repeat (for dext. demo) updated joint positions.

### CODE REVIEW

File: scripts/testLeapGrasp.py
Comment (sjmock99) [inline scripts/testLeapGrasp.py:1]: please change this name to test_leap_grasp.py

File: config/default_params.yaml
Comment (sjmock99) [inline config/default_params.yaml:1]: please put the start_pos as an array, and not individual numbers...

File: scripts/testLeapGrasp.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        # # self.grab_angles[[3, 7, 11]] = 30.0     #15
+
+
+        # self.grab_angles[12] = 100.0
+        # self.grab_angles[13] = 3.0
+        # self.grab_angles[14] = -10.0     #Thumb grip
+        # self.grab_angles[15] = 60.0     #60
+        # # self.grab_angles[15] = 85.0     
+        
+        self.grab_angles[0] = 5.0
```
Comment (sjmock99) [inline scripts/testLeapGrasp.py]: When testing, I dont believe these angles to be correct with the renumbering of the motors

File: config/default_params.yaml [hunk 2]
```diff
@@ -1,11 +1,27 @@
-hand_name: "dom"
 baudrate: 4000000
```
Comment (sjmock99) [inline config/default_params.yaml]: I believe this should be 3Mbps

Comment (Mumbzzz) [reply config/default_params.yaml]: done

File: scripts/testLeapGrasp.py
Comment (Mumbzzz) [reply scripts/testLeapGrasp.py:1]: done

File: scripts/testLeapGrasp.py (hunk 1 above)
Comment (Mumbzzz) [reply scripts/testLeapGrasp.py]: fixed

File: config/default_params.yaml
Comment (Mumbzzz) [reply config/default_params.yaml:1]: done

**Review by sjmock99 — APPROVED** [review]
Looks good to me for now!

**Pull request merged**